openpdf/src/main/java/com/lowagie/text/StandardFonts.java

```diff
@@ -8,37 +8,116 @@ public enum StandardFonts {
     COURIER_ITALIC(Font.COURIER, Font.ITALIC),
     COURIER_BOLD(Font.COURIER, Font.BOLD),
     COURIER_BOLDITALIC(Font.COURIER, Font.BOLDITALIC),
+    // Liberation Mono - Courier compatible
+    /**
+     * @deprecated Use Liberation
+     */
+    @Deprecated
+    LIBERATION_MONO("font-fallback/LiberationMono-Regular.ttf"),
+    /**
+     * @deprecated Use Liberation
+     */
+    @Deprecated
+    LIBERATION_MONO_ITALIC("font-fallback/LiberationMono-Italic.ttf"),
+    /**
+     * @deprecated Use Liberation
+     */
+    @Deprecated
+    LIBERATION_MONO_BOLD("font-fallback/LiberationMono-Bold.ttf"),
+    /**
+     * @deprecated Use Liberation
+     */
+    @Deprecated
+    LIBERATION_MONO_BOLDITALIC("font-fallback/LiberationMono-BoldItalic.ttf"),
```

Comment:
Liberation-Fonts were added in OpenPDF, so these can be removed.

Reply:
Ah ok didn't know. Reverted.